Makefile

```diff
@@ -121,17 +130,18 @@ target=$1
 os=$2
 
 OS_LOWER-$(target)=$(shell echo $(os) | tr '[:upper:]' '[:lower:]')
+TRIPLE_OS-$(target)=$$(TRIPLE_OS-$(os))
```

Comment:
This shouldn't be needed - you can just used TRIPLE_OS-$(os).

It's also worth checking whether OS_LOWER-$(target) is needed any more.